devTools/sbx/build-image.sh

```
@@ -0,0 +1,57 @@
+
+if ! command -v container-structure-test >/dev/null 2>&1; then
+    cat >&2 <<'EOF'
+container-structure-test is required to verify the Docker sandbox image, but it was not found in PATH.
+
+https://github.com/GoogleContainerTools/container-structure-test
+EOF
+    exit 1
+fi
+
+set -x
+container-structure-test test --image=${IMAGE_REF} --config=${SCRIPT_DIR}/test.yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)